draft-ietf-quic-multipath.md

```diff
@@ -625,6 +628,41 @@ delay will most of the time relate to the path with the shortest latency.
 To collect ACK delays on all the paths, hosts should rely on time stamps
 as described in {{QUIC-Timestamp}}.
 
+### ECN and Zero-Length CID Considerations
+
+ECN feedback in QUIC is provided based on counters in the ACK frame
+(see {{Section  19.3.2. of QUIC-TRANSPORT}}). These counters are specific
+to a number space. When sending to zero-length CID receivers, the same number space is
+used for multiple paths, and the ECN feedback cannot unambiguously be assigned to
+a path.
+
+If the sender marks its packets with the ECN capable flags, the network
+will expect standard reactions to ECN marks, such as slowing down
+transmission on the sending path. As the sending path is ambiguous,
+the sender must treat a CE marking as a congestion signal on all sending
+paths.
```

Comment:
```suggestion
transmission on the sending path. If zero-length CID is used, 
the sending path is however ambiguous. Therefore, the sender MUST
treat a CE marking as a congestion signal on all sending paths.
```

Reply:
Note that this change is not only editorial as I added the clause that you only MUST decrease those paths where you actually got a packet acknowledged. This is slightly more complicated but you should have saved the information where which packet was sent anyway and this allows for optimisation in the ACK creation. Given this is a MUST, I think we need this clause.